tests/osaka/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py

```diff
@@ -118,6 +122,24 @@ def test_all_opcodes_in_container(
     )
 
 
+@pytest.mark.parametrize(
+    "opcode",
+    sorted(invalid_eof_opcodes | undefined_opcodes),
+)
+def test_invalid_opcodes_after_stop(
+    eof_test: EOFTestFiller,
+    opcode: Opcode,
+):
+    """
+    Test that an invalid opcode placed after STOP (terminating instruction) invalidates EOF.
+    """
+    eof_code = Container(sections=[Section.Code(code=Op.STOP + opcode), Section.Data("00" * 32)])
```

Comment:
this could be parametrized with all terminating instructions and an `RJUMP[-3]` too

Reply:
+1

Reply:
Added.